client/src/Pages/Home.js

```diff
@@ -11,6 +11,13 @@ const Home = () => {
       });
   }, []);
 
+  function checkShit(e) {
+    e.preventDefault()
+    console.log("what is thiss?!?!?")
+  }
+
+  // Math.floor(Math.random(post) * 4)
```

Comment:
Targeting some clean up
```suggestion
```

Reply:
oops forgot bout that. lol! shall delete